tests/check_examples.py

```diff
@@ -0,0 +1,29 @@
+import json
+import unittest
+from pathlib import (
+    Path,
+)
+
+from dpgen2.entrypoint.args import (
+    normalize,
+)
+
+p_examples = Path(__file__).parent.parent / "examples"
+
+input_files = (
+    p_examples / "almg" / "input.json",
+    # p_examples / "almg" / "input-v005.json",
+    # p_examples / "almg" / "dp_template.json",
+    p_examples / "ch4" / "input_dist.json",
+    # p_examples / "chno" / "dpa_manyi.json",
+    p_examples / "chno" / "input.json",
```

Comment:
Three example files are broken.

Reply:
input-v005.json should be removed because it is written in the deprecated format. 
dp_template.json and dpa_manyi.json are templates for dp training.